base/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="KinEcosystemNoActionBar" parent="Theme.AppCompat.Light.NoActionBar">
+        <item name="colorPrimary">@color/kinecosystem_white</item>
+        <item name="colorPrimaryDark">@color/kinecosystem_transparent</item>
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+    </style>
+
+    <style name="KinEcosystemTranslucent" parent="KinEcosystemNoActionBar">
+        <item name="android:windowBackground">@color/kinecosystem_transparent</item>
+        <item name="android:windowIsTranslucent">true</item>
+        <item name="android:backgroundDimEnabled">false</item>
+    </style>
+
+    <style name="KinEcosystem.Light" parent="KinEcosystemNoActionBar">
+        <item name="primaryTextColor">@color/kinecosystem_title</item>
+        <item name="kinBackgroundColor">@color/kinecosystem_background_light</item>
+        <item name="android:windowBackground">@color/kinecosystem_background_light</item>
+        <item name="android:windowIsTranslucent">false</item>
+        <item name="buttonDisabledColor">@color/kinecosystem_button_disabled_light</item>
+    </style>
+
+    <style name="KinEcosystem.Dark" parent="KinEcosystemNoActionBar">
+        <item name="primaryTextColor">@color/kinecosystem_white</item>
+        <item name="kinBackgroundColor">@color/kinecosystem_background_dark</item>
+        <item name="android:windowBackground">@color/kinecosystem_background_dark</item>
+        <item name="android:windowIsTranslucent">false</item>
+        <item name="buttonDisabledColor">@color/kinecosystem_button_disabled_dark</item>
+    </style>
+
+    <style name="KinEcosystemTranslucent.Light" parent="KinEcosystemTranslucent">
+        <item name="primaryTextColor">@color/kinecosystem_title</item>
+        <item name="kinBackgroundColor">@color/kinecosystem_background_light</item>
+        <item name="buttonDisabledColor">@color/kinecosystem_button_disabled_light</item>
+    </style>
+
+    <style name="KinEcosystemTranslucent.Dark" parent="KinEcosystemTranslucent">
+        <item name="primaryTextColor">@color/kinecosystem_white</item>
+        <item name="kinBackgroundColor">@color/kinecosystem_background_dark</item>
+        <item name="buttonDisabledColor">@color/kinecosystem_button_disabled_dark</item>
+    </style>
+
+
+    <style name="KinecosysTextBase">
+        <item name="android:textColor">?attr/primaryTextColor</item>
+        <item name="android:ellipsize">end</item>
+    </style>
+
+    <!--Title-->
+    <style name="KinecosysTitle" parent="KinecosysTextBase">
+        <item name="android:textSize">@dimen/kinecosystem_title_size</item>
+    </style>
+
+    <style name="KinecosysTitleSmall" parent="KinecosysTextBase">
```

Comment:
KinecosysTitle.Small  as i see it seprate the new attr with dots

Reply:
I agree with you but currently, it will make a lot of changes so I'll do it another time.